### ISSUE TITLE
Declrations of SASS, Less variables being linted

### ISSUE THREAD

**Issue by AndyOGo** [AndyOGo/stylelint-declaration-strict-value#2]
Those should be excluded:

```sh
scss/style/variables.scss
 26:1  ✖  Expected variable, function or keyword for "#103184" of "$color-blue"                         scale-unlimited/declaration-strict-value
 27:1  ✖  Expected variable, function or keyword for "#ff1821" of "$color-red"                          scale-unlimited/declaration-strict-value
 31:1  ✖  Expected variable, function or keyword for "#5c91c0" of "$color-light-blue--light"            scale-unlimited/declaration-strict-value
 32:1  ✖  Expected variable, function or keyword for "#0062a9" of "$color-light-blue"                   scale-unlimited/declaration-strict-value
 33:1  ✖  Expected variable, function or keyword for "#7a9f35" of "$color-green"                        scale-unlimited/declaration-strict-value
 34:1  ✖  Expected variable, function or keyword for "#dedee2" of "$color-blue-gray--light"             scale-unlimited/declaration-strict-value
 35:1  ✖  Expected variable, function or keyword for "#79838d" of "$color-blue-gray"                    scale-unlimited/declaration-strict-value
 37:1  ✖  Expected variable, function or keyword for "#fff" of "$color-white"                           scale-unlimited/declaration-strict-value
 38:1  ✖  Expected variable, function or keyword for "#efefef" of "$color-gray--lighter"                scale-unlimited/declaration-strict-value
 39:1  ✖  Expected variable, function or keyword for "#e5e5e5" of "$color-gray--light"                  scale-unlimited/declaration-strict-value
 40:1  ✖  Expected variable, function or keyword for "#dadada" of "$color-gray--mid"                    scale-unlimited/declaration-strict-value
 41:1  ✖  Expected variable, function or keyword for "#b0b0b0" of "$color-gray"                         scale-unlimited/declaration-strict-value
 42:1  ✖  Expected variable, function or keyword for "#919191" of "$color-gray--dark"                   scale-unlimited/declaration-strict-value
 43:1  ✖  Expected variable, function or keyword for "#6f6f6f" of "$color-gray--darker"                 scale-unlimited/declaration-strict-value
 44:1  ✖  Expected variable, function or keyword for "#666" of "$color-gray--predarkest"                scale-unlimited/declaration-strict-value
 45:1  ✖  Expected variable, function or keyword for "#333" of "$color-gray--darkest"                   scale-unlimited/declaration-strict-value
 46:1  ✖  Expected variable, function or keyword for "#000" of "$color-black"                           scale-unlimited/declaration-strict-value
 50:1  ✖  Expected variable, function or keyword for "#f00" of "$color-red-error"                       scale-unlimited/declaration-strict-value
 54:1  ✖  Expected variable, function or keyword for "#a5afcb" of "$color-blue-logan"                   scale-unlimited/declaration-strict-value
 55:1  ✖  Expected variable, function or keyword for "#eff4f9" of "$color-blue-black-squeeze"           scale-unlimited/declaration-strict-value
 56:1  ✖  Expected variable, function or keyword for "#dde8f2" of "$color-blue-black-squeeze--darker"   scale-unlimited/declaration-strict-value
 57:1  ✖  Expected variable, function or keyword for "#12369d" of "$color-blue-torea-bay"               scale-unlimited/declaration-strict-value
 58:1  ✖  Expected variable, function or keyword for "#e0e0e0" of "$color-gray-alto"                    scale-unlimited/declaration-strict-value
 59:1  ✖  Expected variable, function or keyword for "#22336a" of "$color-blue-astronaut"               scale-unlimited/declaration-strict-value
 60:1  ✖  Expected variable, function or keyword for "#040b20" of "$color-blue-black-pearl"             scale-unlimited/declaration-strict-value
 61:1  ✖  Expected variable, function or keyword for "#131f44" of "$color-blue-bunting"                 scale-unlimited/declaration-strict-value
 62:1  ✖  Expected variable, function or keyword for "#f3f6fc" of "$color-blue-link-water"              scale-unlimited/declaration-strict-value
 63:1  ✖  Expected variable, function or keyword for "#4f5d88" of "$color-blue-blue-bayoux"             scale-unlimited/declaration-strict-value
 64:1  ✖  Expected variable, function or keyword for "#f2f2f2" of "$color-gray-concrete"                scale-unlimited/declaration-strict-value
 65:1  ✖  Expected variable, function or keyword for "#dfdfdf" of "$color-gray-alto--darker"            scale-unlimited/declaration-strict-value
 66:1  ✖  Expected variable, function or keyword for "#1762a5" of "$color-blue-fun"                     scale-unlimited/declaration-strict-value
 67:1  ✖  Expected variable, function or keyword for "#010943" of "$color-blue-stratos"                 scale-unlimited/declaration-strict-value
 68:1  ✖  Expected variable, function or keyword for "#babfd3" of "$color-blue-heather"                 scale-unlimited/declaration-strict-value
 69:1  ✖  Expected variable, function or keyword for "#f7f7f7" of "$color-gray-alabaster"               scale-unlimited/declaration-strict-value
 70:1  ✖  Expected variable, function or keyword for "#fcfcfc" of "$color-gray-alabaster--lighter"      scale-unlimited/declaration-strict-value
 71:1  ✖  Expected variable, function or keyword for "#ccc" of "$color-gray-silver"                     scale-unlimited/declaration-strict-value
 72:1  ✖  Expected variable, function or keyword for "#bcd9f4" of "$color-blue-tropical-blue"           scale-unlimited/declaration-strict-value
```
